### PR TITLE
Add missing `container` notation

### PR DESCRIPTION
**Pull request by tuxiqae**

### CODE REVIEW

File: config/formatting.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ $ docker inspect --format "{{lower .Name}}" container
 
 {% raw %}
 ```console
-$ docker inspect --format '{{split .Image ":"}}'
+$ docker inspect --format '{{split .Image ":"}}' container
```
Comment (dvdksn) [inline config/formatting.md:87]: Just a small suggestion to add angle brackets for the container name placeholder to make it appear more like a placeholder than a sub-command

```suggestion
$ docker inspect --format '{{split .Image ":"}}' <container>
```

Comment (tuxiqae) [reply config/formatting.md:87]: I wholeheartedly agree, just bear in mind that those changes should be made to the rest of the examples as well and not only to these one.

Comment (dvdksn) [reply config/formatting.md:87]: oh yeah, I didn't check the rest of the doc. We can merge it like this and I can take the brackets in a follow-up.

Comment (tuxiqae) [reply config/formatting.md:87]: Should be quite quick to implement, want me to just fix it all in this PR?

Comment (dvdksn) [reply config/formatting.md:87]: sure, if you dont mind! thank you!

**Review by dvdksn — APPROVED** [review]
Bringing this one in, and then we can fix the placeholder style in a followup

**Pull request merged**